src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -43,6 +48,15 @@ private protected DataViewType(Type rawType)
     /// <summary>
     /// The abstract base class for all non-primitive types.
     /// </summary>
+    /// <remarks>
+    /// This class stands in constrast to <see cref="PrimitiveDataViewType"/>. As that class is defined
+    /// to encapsulate cases where instances of the reference type can be freely copied without concerns
+    /// about ownership, mutability, or dispoal, this is defined for those types that are more disposed.
```

Comment:
`dispoal`

Reply:
`that are more disposed.`? What does that mean?

---
In reply to: [274482173](https://github.com/dotnet/machinelearning/pull/3288#discussion_r274482173) [](ancestors = 274482173)

Reply:
I think I was trying to say that one or more of these were a concern, or something along those lines...